tests/parametric/test_library_tracestats.py

```diff
@@ -26,8 +26,7 @@ def _human_stats(stats: V06StatsAggr) -> str:
 
 def enable_tracestats(sample_rate: Optional[float] = None) -> Any:
     env = {
-        "DD_TRACE_STATS_COMPUTATION_ENABLED": "1",  # reference, dotnet, python
```

Comment:
(non-blocking nit) it would be great if we can branch depending on the go package version the tests are run against. This will help if we want to run the tests against an older version of the package. It also helps with documenting what version added the feature.

We're starting to try to do this as of https://github.com/DataDog/system-tests/pull/1442.

We should be able to do it here by doing something like:

```
from utils import context
...

if context.library == "golang" and context.library.version < "x.y.z":
   env["DD_TRACE_FEATURES"] = "discovery"
```

Reply:
Great idea, thanks! And thanks for contributing https://github.com/DataDog/system-tests/pull/1442 💯